src/components/vm/filesystems/vmFilesystemsCard.tsx

```diff
@@ -39,47 +39,45 @@ import { ModalError } from 'cockpit-components-inline-notification.jsx';
 import { FileAutoComplete } from "cockpit-components-file-autocomplete.jsx";
 import { useDialogs } from 'dialogs.jsx';
 
-import { domainCreateFilesystem, domainDeleteFilesystem, domainSetMemoryBacking } from "../../../libvirtApi/domain.js";
+import { virtXmlAdd, virtXmlRemove, virtXmlEdit } from "../../../libvirtApi/domain.js";
 import { vmId } from "../../../helpers.js";
 import { DeleteResourceButton } from '../../common/deleteResource.jsx';
 import { InfoPopover } from '../../common/infoPopover.jsx';
 
 const _ = cockpit.gettext;
 
 export const VmFilesystemsCard = ({
-    connectionName,
-    vmName,
-    vmState,
-    filesystems
+    vm,
```

Comment:
I've been guilty of this pattern myself, but I think we need to be a bit careful here. The whole `vm` object changes very very often due to the usage metrics, no? So passing the whole `vm` down all these components causes a lot more renders now?